docs/language/api.rst

```diff
@@ -947,19 +947,24 @@ get
 ---
 
 ``get`` is used to access single elements in lists and dictionaries. ``get``
-takes two parameters: the *data structure* and the *index* or *key* of the
-item. It will then return the corresponding value from the dictionary or the
-list. Example usage:
+takes at least two parameters: the *data structure* and the *index* or *key* 
+of the item. It will then return the corresponding value from the dictionary 
+or the list. If multiple *index* or *key* values are provided, they are used
```

Comment:
How about changing all the *in lists and dictionaries* to just *in collections*, since it's a bit simpler and handles more built-in cases (e.g. tuples, strings).